README.md

```diff
@@ -484,6 +484,7 @@ Graphical User Interface section has what you need.
   native C binding. [Various licenses][148].
 * [PlutoVG][590] - A standalone 2D vector graphics library in C [``MIT``][MIT]
 * [SAIL][601] - ⛵ The missing small and fast image decoding library for humans (not for machines) [``MIT``][MIT]
+* [Kinc][622] - A low level SDK for building games and media applications in a portable way. Think SDL, but super-charged. [``Zlib``][Zlib]
```

Comment:
```suggestion
* [Kha][622] - Ultra-portable, high performance, open source multimedia framework. [``Zlib``][Zlib]
```

Reply:
Hi @andy5995, thanks for the review. The issue with linking Kha instead of Kinc is that Kha is actually a higher level framework that uses a C# like language called haxe. The Kha framework uses Kinc. Kinc is the C based framework that enables Kha to support multiple native targets.